modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -406,3 +406,87 @@
 	config_job = "engineering_guard"
 	template_access = list(ACCESS_CAPTAIN, ACCESS_CE, ACCESS_CHANGE_IDS)
 	job = /datum/job/engineering_guard
+
+/obj/item/melee/baton/security/loaded/departmental
+	name = "departmental stun baton"
+	desc = "A stun baton that doesn't operate outside of the department it's assigned to. Can be used outside of a department up to three times before needing to return!"
```

Comment:
```suggestion
	desc = "A stun baton fitted with a departmental area-lock - outside of its department, it only has three charges."
```
(Personally would word it like this, by no means is this a required change)
This change would ideally apply to all the batons' descriptions